fceciliano/TrainingFabian
# Feature/task 9

# Pull Request
## Description
This task consist of build the website of the One Piece, specifically the home page, using the JavaScript library React which allow build user interfaces based on UI components. In this task were applied the knowledge learned in the different resources of the training. Also, was used as support the previous tasks to develop using atomic design.

Feature (Task #9)

## Type of change

## How Has This Been Tested?

To execute the projects you will need:
- Clone the repository
- Search and open the folder: “task09-reactjs/one-piece-page”.
- In a terminal execute the commands:
    - “yarn install” to download all the packages.
    - “yarn start” to execute the project.

To verify this task:
- Resize the browser window and verify the consistency of all the components, also check that the components change following de guidelines of figma
- Check the typography and style used.
- Check the correct use of atomic design 
- Verify that the format of the CSS class names follow the method used in previous tasks.
